cmd/influx/global.go

```diff
@@ -276,3 +276,20 @@ func getBucketFlags(params *clients.BucketParams) []cli.Flag {
 func getOrgBucketFlags(c *clients.OrgBucketParams) []cli.Flag {
 	return append(getBucketFlags(&c.BucketParams), getOrgFlags(&c.OrgParams)...)
 }
+
+// getAuthLookupFlags returns flags used for authorization, requiring either an ID or Username,
+// and can specify to require one but not both.
+func getAuthLookupFlags(params *clients.AuthLookupParams) []cli.Flag {
```

Comment:
I'd also move this to `cmd/influx/v1_auth.go` since it's not useful to other commands